### ISSUE TITLE
About learning_rate

### ISSUE THREAD

**Issue by Brokkoi** [jimmy9704/GPS-GLASS#2]
May I ask what was the learning rate of the discriminator used during training to achieve the highest accuracy?

**Comment by jimmy9704** [comment]
Thank you for your interest in our project. We used the same learning rate of 1e-4 as DANNet.
